testnet/install/linux-ubuntu.md

```diff
@@ -58,15 +58,15 @@ And update the docker.sock user group
 sudo chown root:docker /var/run/docker.sock
 ```
 
-### Install Bazel v6.3.2 (Instructions taken from https://bazel.build/install/ubuntu)
+### Install Bazel v7.5.0 (Instructions taken from https://bazel.build/versions/7.5.0/install/ubuntu)
 
 ```
 sudo apt install apt-transport-https curl gnupg -y
 curl -fsSL https://bazel.build/bazel-release.pub.gpg | gpg --dearmor >bazel-archive-keyring.gpg
 sudo mv bazel-archive-keyring.gpg /usr/share/keyrings
 echo "deb [arch=amd64 signed-by=/usr/share/keyrings/bazel-archive-keyring.gpg] https://storage.googleapis.com/bazel-apt stable jdk1.8" | sudo tee /etc/apt/sources.list.d/bazel.list

```

Comment:
I feel like hardcoding `arch=amd64` is a risk in case a user has another architecture such as `x86_64`.

Reply:
Agree this should be avoided